phoenix-core/src/it/java/org/apache/phoenix/calcite/CalciteIT.java

```diff
@@ -1315,6 +1315,17 @@ public void initTable() throws Exception {
                         {6L, "00A323122312312"},
                         {8L, "00A423122312312"}})
                 .close();
+
```

Comment:
It's nice to add more test case in CalciteIT, but I'm just curious, why aren't existing tests in testSequence() sufficient?